Contribute/index.md

```diff
@@ -47,6 +47,8 @@ Quick edits streamline the process to report and fix small errors and omissions
 4. When you're finished editing, scroll to the bottom of the page. In the **Propose changes** area, enter a title and optionally a description for your changes. The title will be the first line of the commit message. Select **Propose changes** to commit your changes:
 
    :::image type="content" source="media/index/commit-changes.png" alt-text="Propose and commit file changes.":::
+   
+   Note: In other Microsoft documentation, GitHub may require a branch to be created in order to **Propose changes**.
```

Comment:
```suggestion

```

Reply:
@Andrew-Nielsen I think that it'll create a branch every time - mentioning it in the step above could be better than having it as a note.

Reply:
@jehchow, after removing the note, it looks like it kept an extra space. Are you seeing the same thing?